product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_biganimal_observability/index.mdx

```diff
@@ -58,7 +58,13 @@ By default, these charts displays the historical data of the last 15 minutes. To
   - **Active connections** (line chart) &mdash; The current number of connections between the client applications and the Postgres cluster.
   - **Disk throughput** (line chart) &mdash; The amount of data transferred to and from the disk per second for the Postgres cluster.
   - **Database size** (line chart) &mdash; The amount of storage volume used by the Postgres cluster.
+  - **Session wait states** &mdash; These charts are powered by [EDB wait states](https://www.enterprisedb.com/docs/pg_extensions/wait_states/). They include:
+    - **Wait times** (line chart) &mdash; This chart shows the approximate wait times for different wait events in a Postgres database, measured in seconds. It groups each wait event type to make it easy to compare and identify potential system issues.
+    - **Wait events** (line chart) &mdash; This chart visualizes the approximate wait times of wait events in a Postgres database, measured in seconds.
```

Comment:
```suggestion
    - **Wait events** (line chart) &mdash; Visualizes the approximate wait times of wait events in a Postgres database, in seconds.
```
Why "visualizes" rather than "shows"?